examples/vectors-clamptoground.html

```diff
@@ -0,0 +1,32 @@
+<?xml version='1.0' encoding='UTF-8'?>
```

Comment:
I would better not duplicate the vector.js example but enhance it with a button doing:
```js
vectorLayer.set('altitudeMode', 'clampToGround');
vectorLayer2.set('altitudeMode', 'clampToGround');
```